### ISSUE TITLE
a Problem With "Select Area" Tool

### ISSUE THREAD

**Issue by gubuntu** [inasafe/inasafe#4473]
_From @samnawi on April 19, 2017 9:30_

I got a problem after using "Select Area" Tool on Earthquake Realtime.
After I click "remove selection", I can't drag the map to the other particular area.
![realtime2](https://cloud.githubusercontent.com/assets/8871958/25173030/f749b264-251c-11e7-9a58-089d305692c6.gif)

Browser : Chrome

cc : @lucernae @ivanbusthomi 

_Copied from original issue: inasafe/inasafe-realtime#86_

**Comment by gubuntu** [comment]
_From @Charlotte-Morgan on August 8, 2017 7:7_

hi @lucernae  - this might be good to fix before the training

**Comment by gubuntu** [comment]
@lucernae, I also get the following:
1. Select Area always zooms out to multiple earths even if I have manually zoomed in before. Why does it autozoom at all? I'd expect it to let you digitise an area independent of zoom level. 
1. 'Select area within current zoom' seems to just zoom in to one random event and show its popup.
1. There seems to be a strange interplay between the select area tool and other filters (I tried date) - e.g. after create and destroying a select area a few times I filter by date and some spatial filter also gets applied. In addition it zooms into one record and shows its popup. 

I'm not sure if the select area tool is useful in its current state - I think it is a bit confusing and unintuitive and isn't working properly. Was its original intention to spatially filter events?

If it's not a quick fix consider disabling it for the training?

I'm on Firefox 52 on Mac.
